java/web/src/main/resources/proxy.properties

```diff
@@ -27,4 +27,4 @@ reqtypeWhitelist.capabilities = .*[Gg]et[Cc]apabilities.*
 reqtypeWhitelist.featureinfo = .*[Gg]et[Ff]eature[Ii]nfo.*
 reqtypeWhitelist.csw = .*csw.*
 reqtypeWhitelist.geostore = .*geostore.*
-reqtypeWhitelist.generic = (.*exist.*)|(.*pdf.*)|(.*map.*)|(.*wms.*)|(.*wmts.*)|(.*tms.*)|(.*wfs.*)|(.*ows.*)|(.*spatialreference.*)|(.*wps.*)|(.*tileset\.json.*)|(.*\.b3dm.*)|(.*\.i3dm.*)|(.*\.pnts.*)
+reqtypeWhitelist.generic = (.*exist.*)|(.*pdf.*)|(.*map.*)|(.*wms.*)|(.*wmts.*)|(.*tms.*)|(.*wfs.*)|(.*ows.*)|(.*spatialreference.*)|(.*wps.*)|(.*rest.*)|(.*tileset\.json.*)|(.*\.b3dm.*)|(.*\.i3dm.*)|(.*\.pnts.*)
```

Comment:
please keep aligned also this file

project\standard\templates\web\src\main\resources\proxy.properties